### PR TITLE
shower temperature adj hint

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

span_notice("You can <b>alt click</b> to change the temperature.")

# Changelog

:cl:  
rscadd: Added a hint so newer players can figure out how to change a shower's temperature
/:cl:

### CODE REVIEW

**Comment by tattax** [comment]
why not add it to description

**Comment by ynot01** [comment]
blue notices at the end of examine are typically used for instructions & hints

**Comment by tattax** [comment]
you could technically do that in the desc but that's just a nitpick

**Comment by ynot01** [comment]
> you could technically do that in the desc but that's just a nitpick

this would put it above other examine modifiers like "It is slightly damaged." which would look weird compared to other instances of similar hint text

**Review by Bibby0110 — APPROVED** [review]

File: code/game/objects/structures/shower.dm [hunk 1]
```diff
@@ -65,6 +65,10 @@
 	handle_mist()
 	return TRUE
 
+/obj/machinery/shower/examine(mob/user)
+	. = ..()
+	. += span_notice("You can <b>alt click</b> to change the temperature.")
```
Comment (Bibby0110) [inline code/game/objects/structures/shower.dm]:
```suggestion
	. += span_notice("You can <b>alt-click</b> to change the temperature.")
```

**Pull request merged**